src/twitter/Timeline.js

```diff
@@ -357,24 +390,32 @@ class Timeline {
       log.info("Timeline", "Poll request hit count limit. Request likely incomplete.");
     }
     this.twitter.storage.set_since(sinceId, results[0].id_str);
-    const shouldForceUserId = this.is_feed_exceeding_user_limit(results, isTimeline);
+    const hotRooms = this.roomsExceedingJoinThreshold(results, feed.room, isTimeline);
```

Comment:
I find it confusing that `feed.room` is actually an array. Perhaps `feed.rooms`?